src/mixins/commands_mixin.js

```diff
@@ -406,6 +406,8 @@ export const commands_mixin = {
         image_type: this.camera_image_type,
       }
       let opt_params = {
+        #object: this.object_name,
```

Comment:
Just an fyi, javascript comments are denoted by double backslashes for single lines or enclosed by backslash and star for longer ones.  :)
```javascript
// here is a single line comment

/* 
here is a 
multiline comment
*/
'''